src/picknik_ur_multi_arm_gazebo_config/description/apriltags/README.md

```diff
@@ -0,0 +1,2 @@
+The AprilTag mesh and image files come from the following repo:
+https://github.com/TAMS-Group/tams_apriltags
```

Comment:
Can we reuse the april tag files from the "simple" gazebo studio config? Maybe consider making this config depend on it to reuse the files if that is possible?